File: 2304-cells-in-a-range-on-an-excel-sheet/2304-cells-in-a-range-on-an-excel-sheet.java
class Solution {
    public List<String> cellsInRange(String s) {
      char initCol = s.charAt(0), finalCol = s.charAt(3);
      int initRow = Character.getNumericValue(s.charAt(1)), finalRow = Character.getNumericValue(s.charAt(4));
      List<String> list = new ArrayList<String>();
      for(char i=initCol; i<=finalCol; i++){
        for(int j=initRow; j<=finalRow; j++){
          list.add(i+String.valueOf(j));
        }
      }
      return list;
    }
}

File: 2304-cells-in-a-range-on-an-excel-sheet/2304-cells-in-a-range-on-an-excel-sheet.js
/**
 * @param {string} s
 * @return {string[]}
 */
var cellsInRange = function(s) {
  const initCol = s.charCodeAt(0), finalCol = s.charCodeAt(3);
  const initRow = Number(s[1]), finalRow = Number(s[4]);
  const arr = [];
  for(let i=initCol; i<=finalCol; i++){
    const colLetter = String.fromCharCode(i);
    for(let j=initRow; j<=finalRow; j++){
      arr.push(`${colLetter}${j}`);
    }
  }
  return arr;
};